modules/microshift-config-yaml.adoc

```diff
@@ -27,20 +29,37 @@ node:
   hostnameOverride: "" <5>
   nodeIP: "" <6>
 apiServer:
-  subjectAltNames: [] <7>
+  advertiseAddress:
+    - 10.44.0.0 <7>
```

Comment:
This line needs to be joined with lined 32 (the line above). Please remove the '-' preceding the IP address.